### ISSUE TITLE
Failing to decode image

### ISSUE THREAD

**Issue by arisbasic** [uclouvain/openjpeg#789]
I have an jpeg2000 image that i cant decode with openjpeg (git)
aware jpeg2000 library can decode it fine. not sure what library was used to compress it.

im just using openjp2 portion. 

http://www.filedropper.com/img_6

Thx

**Comment by malaterre** [comment]
You are missing the EOF (ffd9). That is all. OpenJPEG is a bit more picky than kdu and other, but this is an easy fix on your side.

**Comment by arisbasic** [comment]
anyway to add that to openjpeg ? is this on the end of the file (stream) ?

On Thu, Jun 2, 2016 at 12:12 PM, Mathieu Malaterre <notifications@github.com

**Comment by malaterre** [comment]
well I was about to ask the opposite. Could you share the original DICOM file instead ?

**Comment by arisbasic** [comment]
here it is

On Thu, Jun 2, 2016 at 12:58 PM, Mathieu Malaterre <notifications@github.com

**Comment by malaterre** [comment]
I do not believe attachement works in github issue tracker. You need to share it using an alternative mechanism
